spec/author_module_spec.rb

```diff
@@ -0,0 +1,138 @@
+require_relative '../module/author_module'
```

Comment:
![image](https://github.com/uno36/ruby-capestone/assets/93039359/e452fb71-a54a-46c8-b11d-df8cf6cc3bc7)

- [ ] Great so far but it would be nice if you check this error which i am getting on running the tests of your app :fearful: Kindly fix them to make your app tested perfectly :+1: